lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -108,14 +68,16 @@ function filterOutArtifacts(result) {
  * @param {!Connection} connection
  * @param {string} url
  * @param {!Object} options Lighthouse options.
- * @param {!Object<boolean>} requestedAggregations Names of aggregations to include.
+ * @param {!Object<boolean>} aggregationTags Ids of aggregation tags to include.
  * @return {!Promise}
  */
-window.runLighthouseForConnection = function(connection, url, options, requestedAggregations) {
+window.runLighthouseForConnection = function(connection, url, options, aggregationTags) {
   // Always start with a freshly parsed default config.
   const runConfig = JSON.parse(JSON.stringify(defaultConfig));
 
-  filterConfig(runConfig, requestedAggregations);
+   // Change tags object to a plain array of tag strings
+  const chosenTags = aggregationTags.filter(tag => tag.value).map(tag => tag.id);
+  Config.rebuildConfigFromTags(runConfig, chosenTags);
```

Comment:
Only downside is that you might now want that logic of rebuildConfig inside your constructor but in my opinion it's not something we want to expose to the world